### ISSUE TITLE
backslash escaping issue

### ISSUE THREAD

**Issue by CheyiLin** [influxdata/influxdb-ruby#200]
Hi,

This issue is from fangli/fluent-plugin-influxdb#80, it seems that the point data value contains double backslashes will cause write error or incorrect escaping.

The point data hash:
```
{:timestamp=>1501729298688, :series=>"system.filesystem", :values=>{"v.device_name"=>"D:\\", "v.mount_point"=>"D:\\", "v.total"=>26840395776, "v.free"=>5271150592, "v.available"=>5271150592, "v.files"=>0, "v.free_files"=>0, "v.used.pct"=>0.8036, "v.used.bytes"=>21569245184}, :tags=>{"hostname"=>"my-machine"}}
```

And the result:
```
> SELECT "v.device_name", "v.mount_point" FROM "mydb"."system.filesystem"

time    v.device_name             v.mount_point
----    ----                      ----
....    D:",v.mount_point="D:\    D:
```

**Comment by dmke** [comment]
Oh, that's definitely a bug. I will have a look when I'm back at my dev machine.

**Comment by dmke** [comment]
I've fixed the bug. Will release v0.4.2 after Travis has run.

**Comment by dmke** [comment]
Here you go: https://rubygems.org/gems/influxdb/versions/0.4.2

Thanks again for reporting this!

**Comment by CheyiLin** [comment]
Thank you for fixing this quickly. Cheers!

**Comment by maxp-edcast** [comment]
Hi I am using `InfluxDB vv1.4.1 (git: master 882ce8753d52d04a92c01758d8efcfe5c37bd93a)`

and have the following behavior:

```
INFLUXDB_CLIENT = InfluxDB::Client.new(config)
INFLUXDB_CLIENT.write_point "things", {
  time: Time.now.to_i,
  tags: { foo: "\\", },
  values: { bar: 0 }
}
# => InfluxDB::Error: {"error":"unable to parse 'asd,foo=\\ bar=0i': invalid tag format"}
```

I'm worried that this can constitute a potential injection risk for those who don't know to escape their backslashes. I don't have a reproducible example, but regardless, I think it would be helpful if this gem would parameterize the input before sending to influx. 

The issue only happens when the backslash is at the end of the string. `tags: { foo: "\\1" }` works fine.

I am hoping you can advise on how to resolve these questions. Do you think this may introduce injection risk? Is this the expected behavior for pushing `"\\"`? Has it been fixed since v1.4.1? I would potentially be available to make a PR, but I'd like to discuss it first.
